styles.scss

```diff
@@ -845,6 +833,32 @@ q {
     }
   }
 }
+.AddComponentFormWrapper {
+  border-bottom: $list-group-border-width solid $list-group-border-color;
+  display: grid;
+  grid-gap: 5px 10px;
+  grid-template-columns: 100px 1fr 1fr auto 120px 31px;
+  grid-template-areas:
+    "sharesTotal addComponentButton addComponentButton    addComponentButton    addComponentButton     addComponentButton";
```

Comment:
Nit: the multiple spaces separators were there to align the blocks, which isn't applicable anymore